src/loaders/compute_tools/tool_common.py

```diff
@@ -148,21 +156,20 @@ def _parse_args(self):
 
         # Required flag arguments
         required.add_argument(
-            f'--{loader_common_names.LOAD_VER_ARG_NAME}', required=True, type=str,
-            help=loader_common_names.LOAD_VER_DESCR
+            f"--{loader_common_names.ENV_ARG_NAME}", required=True, type=str,
+            choices=loader_common_names.KB_ENV + [loader_common_names.DEFAULT_ENV],
```

Comment:
Should this default to PROD like the workspace downloader or be required, do you think? The former would be less work for users but easier to accidentally use the wrong environment so I'm not really sure.

Reply:
👍  making PROD as default.

Reply:
:+1: